packages/zowe-explorer/CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ### New features and enhancements
 
+- Added Time Started, Time Ended, and Time Submitted job properties to the Jobs table view. [#3055](https://github.com/zowe/zowe-explorer-vscode/issues/3055)
+- Implemented copy/paste functionality of data sets within and across LPARs. [#3012](https://github.com/zowe/zowe-explorer-vscode/issues/3012)
+
+### Bug fixes
+
```

Comment:
Can we move the following entry (below) here?

- Fixed issue where the 'Delete' key binding for the USS tree returns a 'contextValue' error. [#2796](https://github.com/zowe/zowe-explorer-vscode/issues/2796)

Reply:
Resolved in 21ce399cbd127964ec311fef6f5245ec90940bc3